teeracle/src/mock.rs

```diff
@@ -49,11 +49,11 @@ frame_support::construct_runtime!(
 		NodeBlock = Block,
 		UncheckedExtrinsic = UncheckedExtrinsic,
 	{
-		System: frame_system::{Pallet, Call, Config, Storage, Event<T>},
-		Balances: pallet_balances::{Pallet, Call, Storage, Config<T>, Event<T>},
-		Timestamp: timestamp::{Pallet, Call, Storage, Inherent},
-		Teerex: pallet_teerex::{Pallet, Call, Storage, Event<T>},
-		Exchange: pallet_exchange::{Pallet, Call, Storage, Event<T>},
+		System: frame_system,
+		Balances: pallet_balances,
+		Timestamp: timestamp,
+		Teerex: pallet_teerex,
+		Teeracle: pallet_teeracle,
```

Comment:
Nice. Out of personal interest. Can you elaborate on this?

Reply:
Before the `construct_runtime!` macro needed you to specify the types you wanted to build for each pallet such as `Call`, `Event` etc. 

However now that is unneeded. The macro can scan and see which types to build from the pallet definition itself. The most you might want is just to number each pallet i.e. `System: frame_system = 1,` etc.